db/models/account.py

```diff
@@ -1,9 +1,18 @@
+import enum
 import uuid  # noqa
 
 from db import db
 from sqlalchemy_utils.types import UUIDType  # noqa
 
 
+class Role(enum.Enum):
+    ADMIN = "Admin"
+    LEAD_ASSESSOR = "Lead Assessor"
```

Comment:
Are we sure this style works on Postgres? I would think it needs the integers for the values internally>

Reply:
I don't think Postgres cares about or uses the Python object value -- only as the migration in alembic just uses the names `        role = postgresql.ENUM(
            "ADMIN",
            "LEAD_ASSESSOR",
            "ASSESSOR",
            "COMMENTER",
            "APPLICANT",
            name="role",
        )
        role.create(op.get_bind())`